source/assets/stylesheets/_index.scss

```diff
@@ -0,0 +1,95 @@
+.index {
+  @include outer-container($medium-screen);
+  background: linear-gradient(180deg, white 25%, tint($husker-du, 80%));
+  color: $base-font-color;
+  padding: $base-spacing;
+  text-align: center;
+  height: 100%;
```

Comment:
I need this b/c 100vh was causing issues with the gradient stopping at the initial browser dimensions.